pypesto/visualize/parameters.py

```diff
@@ -121,6 +121,17 @@ def scale_parameters(x):
             start_indices=start_indices,
             plot_inner_parameters=plot_inner_parameters,
         )
+
+        # parse fvals and parameters
+        fvals = np.array(fvals)
+        # remove nan or inf values
+        xs, fvals = delete_nan_inf(
+            fvals=fvals,
+            x=xs,
+            xdim=len(ub) if ub is not None else 1,
+            magnitude_bound=WATERFALL_MAX_VALUE,
+        )
+
```

Comment:
Moving the `delete_nan_inf` to here makes it so the plotting of reference points doesn't go through it. Is that an issue? I've never used reference points, but they also possibly might contain infinite/nan values, no?

Reply:
Good point, it's not a problem though, nans are just not shown in the line plot.